lib/agenda.js

```diff
@@ -76,6 +90,7 @@ Agenda.prototype.database = function(url, collection, options, cb) {
   MongoClient.connect(url, options, function( error, db ){
     if (error) {
       if (cb) {
+        debug('connected to %')
```

Comment:
BTW just in case someone thinks of adding `url` to that debug statement:
I'd be hesitant to log `url` here since it might contain user+pass.